otwcode/otwarchive
# Ao3 4778 challenge prompt meme strong params

# Pull Request Checklist


## Issue

https://otwarchive.atlassian.net/browse/AO3-4778

## Purpose

What does this PR do?

Adds strong_params to PromptMeme model and PromptMemesController

## Testing

How can the Archive's QA team verify that this is working as you intended? (If you have access, please copy this into the JIRA ticket for them!)

1. Log in
2. Browse > Collections > New Collection
3. Fill in required information and choose "Prompt Meme" for "Type of challenge, if any"
4. Submit
5. Fill in form with whatever combination of settings you like
6. Submit
7. Choose "Challenge Settings" from side bar
8. Edit form with whatever combination of settings you like
9. Update

## References

Are there any other relevant issues / PRs / mailing lists discussions related to this?

## Credit

*What name do you want us to use to credit your work in the [Archive of Our Own's Release Notes](http://archiveofourown.org/admin_posts?tag=1)?*

Wendy

*What pronouns do you prefer (she/her, he/him, zie/hir etc)?*

She/her
